HackingWithSwift/SwiftUI/Project2/Guess the Flag/Guess the Flag/ContentView.swift

```diff
@@ -17,8 +32,9 @@ struct ContentView: View {
     @State private var endGame: Bool = false
     @State private var counter: Int = 0
     @State private var finalScore: String = ""
-
-
+    @State private var animationAmount: Double = 0.0
+    @State private var currentIndex: Int? = nil
```

Comment:
There’s no need to provide an initial value if the property is an optional type.